docs/modules/datasources/pages/performance_datasource.adoc

```diff
@@ -11,24 +11,74 @@ The query editor provides the ability to build queries of the following types:
 === Attribute queries
 
 Used to extract data points from time series data that OpenNMS collects and stores.
-The query is used to identify which attribute (or series) is being queried.
+The query is used to identify which attribute(s) (or series) are being queried.
 
 [options="header, autowidth"]
+[cols="1,2,1"]
 |===
-| Name               | Description
-| Node               | Node ID, or FS:FID (foreign source and foreign id) of the node to query.
-                       Can also be a variable with a `nodeFilter()` query.
-| Resource           | Resource ID containing the requested attribute.
-                       Can also be a variable with a `nodeResources()` query.
-| Attribute          | Name of the attribute to query.
-| Sub-Attribute      | Name of the sub-attribute to query.
-                       This can be used to query metrics which are modeled using many datasources in a single RRD file (i.e., StrafePing).
-| Fallback Attribute | Name of the attribute to query if the requested attribute was not available.
-| Aggregation        | RRD aggregation to use when exporting the metrics.
-                       Possible values are `Average`, `Min`, `Max` or `Last`.
-
-| Label              | Name for the series.
-                       Must be unique for all attributes and expression results in the query.
+| Name               
+| Description
+| Allowed Glob Expressions
```

Comment:
While glob expressions are the technical term, as a user manual, IMO, an alternate term may be more appropriate for non-developers.
```suggestion
| Allowed Filter Expressions
```

Reply:
OK